Assets/MixedRealityToolkit/Extensions/StringBuilderExtensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Text;
+
+namespace Microsoft.MixedReality.Toolkit
+{
+    /// <summary>
+    /// <see cref="System.Text.StringBuilder"/> Extensions.
+    /// </summary>
+    public static class StringBuilderExtensions
+    {
+        /// <summary>
+        /// Append new line
+        /// </summary>
+        public static StringBuilder NewLine(this StringBuilder sb)
+        {
+            sb.Append("\n");
```

Comment:
Could you use [Environment.NewLine](https://docs.microsoft.com/en-us/dotnet/api/system.environment.newline?view=netframework-4.8) here?